pkg/test/framework/integration/framework_test.go

```diff
@@ -90,3 +93,49 @@ func TestParallel(t *testing.T) {
 	assertClosedBefore(t, l1a, top)
 	assertClosedBefore(t, l1b, top)
 }
+
+// Validate that cleanup is done synchronously for asynchronous tests
+func TestParallelWhenDone(t *testing.T) {
+	errors := make(chan error, 10)
+	framework.NewTest(t).
```

Comment:
```suggestion
	framework.NewTest(t).Features("TestFramework").
```

Reply:
This coupled with a one line change to features.yaml should make the test pass the whitelist.  Let me know if that is not the case.